rotkehlchen/chain/ethereum/modules/convex/decoder.py

```diff
@@ -192,7 +200,51 @@ def _decode_gauge_events(self, context: DecoderContext) -> DecodingOutput:
                         event.notes = f'Claim {event.balance.amount} {crypto_asset.symbol} reward from convex {self.pools[context.tx_log.address]} pool'  # noqa: E501
                     else:
                         event.notes = f'Claim {event.balance.amount} {crypto_asset.symbol} reward from convex'  # noqa: E501
-        return DecodingOutput(refresh_balances=found_event_modifying_balances)
+        return DecodingOutput(
+            refresh_balances=found_event_modifying_balances,
+            matched_counterparty=matched_counterparty,
+        )
+
+    def _check_lock_withdrawal_funds(
+            self,
+            transaction: 'EvmTransaction',  # pylint: disable=unused-argument
+            decoded_events: list['EvmEvent'],
+            all_logs: list['EvmTxReceiptLog'],
+    ) -> list['EvmEvent']:
+        """
+        Process unlocked CVX withdrawals.
+
+        The withdrawal event need to be handled after all the events get decoded since it is
```

Comment:
typo of what I told you discord. Fix next time if only thing.

```suggestion
        The withdrawal event needs to be handled after all the events get decoded since it is
```

Reply:
adjusted it